serial/serialutil.py

```diff
@@ -217,6 +217,7 @@ def __init__(self,
         self._dtr_state = True
         self._break_state = False
         self._exclusive = None
+        self.logger = None
```

Comment:
This line appears in a list of attributes that are assigned via properties. It should move up above the comment `correct values are assigned below through properties`.